backend/migrations/20240927070412_create_initial_as_tables.sql

```diff
@@ -7,9 +7,15 @@ CREATE TABLE as_batched_keys(
     voprf_server BYTEA NOT NULL
 );
 
+CREATE TYPE aead_ciphertext AS (
```

Comment:
NB: This change will require wiping the server db.

Reply:
Good point. I'll try to remember that when merging.